ui/packages/consul-ui/app/components/code-editor/skin.scss

```diff
@@ -24,7 +24,7 @@ $syntax-dark-gray: #535f73;
   --syntax-yellow: rgb(var(--tone-yellow-500));
 }
 .CodeMirror {
-  max-width: 1150px;
+  max-width: 1260px;
```

Comment:
Oh dear 😆 it was just this 🤣 . Surprising nobody has tweaked this before now 🤔

Reply:
I know, right? It happens, it's always a single line of code that is holding things together or that breaks everything :)